src/main/resources/postfixTemplates/ErrPostfixTemplate/after.rs.template

```diff
@@ -0,0 +1,3 @@
+fn foo(slice: &[i32]) {
```

Comment:
nit: these templates (and some tests) do not compile. You can change the return type to `Result`/`Option` so that there is only the `Err(...)` expression inside the function.

Reply:
Fair point, let me change that 😉

Reply:
I made them simpler - not using slice, just a number and make the function return. I hope it's fine now.